packages/ibm-products-styles/src/components/Coachmark/_coachmark-overlay.scss

```diff
@@ -91,7 +91,7 @@ $draghandle-btn-class: #{$block-class}__handle;
 
   &--bottom {
     $horizontal-push: calc($distance-offset - $caret-center);
-    $top: calc(-1 * $caret-height);
+    $inset-block-start: calc(-1 * $caret-height);
```

Comment:
Not sure if this was intended but it doesn't look like this is being used either

Reply:
yes good catch, thanks Matt